src/app/loans/loans-view/loan-account-actions/loan-account-actions.component.html

```diff
@@ -2,5 +2,6 @@
   <mat-card>
     <mifosx-loans-account-close *ngIf="actions.close"></mifosx-loans-account-close>
     <mifosx-undo-approval *ngIf="actions.undo_approval"></mifosx-undo-approval>
+    <mifosx-assign-loan-officer *ngIf="actions.assign_loan_officer" [dataObject]="actionButtonData" ></mifosx-assign-loan-officer>
```

Comment:
camel casing

Reply:
Will change for all in the refactoring PR.